wiki/Performance_Troubleshooting/vi.md

```diff
@@ -0,0 +1,106 @@
+# Khắc phục sự cố hiệu năng trong osu!
+
+osu! luôn cố gắng chạy với thiết lập mặc định và sẽ cảnh báo bạn nếu bạn đang làm/chạy gì đó có thể gây ảnh hưởng tới hiệu năng của trò chơi.
+
+Không may là với phạm vi các phần cứng được sử dụng để chơi osu! là quá lớn, sẽ rất khó để đảm bảo hiệu năng tối ưu cho mọi người chơi. Trang này ngoài việc giúp bạn tự phát hiện vấn đề hiệu năng mình gặp phải, mà còn cung cấp một vài thuật ngữ bạn có thể học (và có thể thực hiện kiểm tra) để giúp chúng tôi giúp được bạn khi bạn cần hỗ trợ.
+
+## Các kiểu "lag"
+
+Rất khó để hỗ trợ bạn khi mọi vấn đề bạn gặp được gộp chung vào một từ "lag", nên hãy bắt đầu bằng việc định nghĩa một vài thuật ngữ cho từng trường hợp cụ thể.
+
+### Âm thanh bị lặp
+
+Nhạc của beatmap thi thoảng bị tua ngược, hoặc cảm thấy giật. Về cơ bản thì khi game lag, bạn có thể cảm nhận thấy nó qua âm thanh nghe được. Trước khi tiến hành chẩn đoán hoặc báo cáo, bạn hãy thử:
+
+- Kiểm tra xem nó có hiện tượng tương tự với mọi thiết bị âm thanh khác (loa, tai nghe, ...). Nếu bạn dùng tai nghe có chân cắm USB, bạn hãy thử kết nối qua analog output của PC.
```

Comment:
```suggestion
- Kiểm tra xem liệu hiện tượng tương tự có xảy ra với mọi thiết bị âm thanh khác (loa, tai nghe, ...). Nếu bạn dùng tai nghe có chân cắm USB, bạn hãy thử kết nối qua cổng ra tương tự của PC.
```

Reply:
I approve the first two. For the `analog output` part, many ppl nowadays using the word "a-na-loóc", which is how to pronounce analog.
Also, many ppl still using Windows English version, so it's kinda best to leave it there, so ppl can find it.
Not that many ppl know analog means `tương tự` though, I insist leaving the terminologies stay the same.

Reply:
Come to think of that...yeah, not many people know what analog is when translated.
```suggestion
- Kiểm tra xem liệu hiện tượng tương tự có xảy ra với mọi thiết bị âm thanh khác (loa, tai nghe, ...). Nếu bạn dùng tai nghe có chân cắm USB, bạn hãy thử kết nối qua analog output của PC.
```